server/src/optima/optimization.py

```diff
@@ -13,7 +13,8 @@
 from flask.ext.login import login_required
 from dbconn import db
 from async_calculate import CalculatingThread, start_or_report_calculation, cancel_calculation, check_calculation
-from utils import check_project_name, project_exists, pick_params, load_model, save_working_model, report_exception
+from utils import check_project_name, project_exists, pick_params, load_model, save_working_model, \
```

Comment:
Nope. working model is only saved within the async thread, and pick_params was not needed in optimization.py. thanks for the reminder about pylint :-}